src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -73,8 +77,9 @@ private static VersionInfo GetVersionInfo()
             return new VersionInfo(
                 modelSignature: "TENSFLOW",
                 //verWrittenCur: 0x00010001, // Initial
-                verWrittenCur: 0x00010002,  // Upgraded when change for multiple outputs was implemented.
```

Comment:
Since this is a brand-new transform, do we really need to increment the version number on every PR? I'm trying to understand the requirements/practice in this space.

Reply:
This is required when there is a change in serialization/deserialization of data in the transform. For example, the version was upgraded to `0x00010002` when multiple outputs were implemented where saving and loading of model was changed a bit. It was changed to `0x00010003` when there was modifications related to saving and loading of un-frozen models.

---
In reply to: [218200584](https://github.com/dotnet/machinelearning/pull/853#discussion_r218200584) [](ancestors = 218200584)

Reply:
My point is that we've only ever released this transform in a single preview release (v0.6). So how are we on v3 of this transform already? Do we really need to maintain backwards compatibility for build-to-build changes in our models? If I make one change today, and then another tomorrow - do I really need to increment the version both times? It would seem to me that we would only need to rev the version across releases. Multiple changes in the same release shouldn't require multiple version changes IMO.

Reply:
Yes, your thinking may be correct. The version `0x00010002` was correctly bumped after the last release. I am not sure if we have to bump it again to `0x00010003` within same release. 

I am not sure if we take into account the intermediate builds as well because it was our practice previously to change it even for nightly builds.


---
In reply to: [218508911](https://github.com/dotnet/machinelearning/pull/853#discussion_r218508911) [](ancestors = 218508911)

Reply:
we decided to keep it at 0x00010002 

---
In reply to: [218518368](https://github.com/dotnet/machinelearning/pull/853#discussion_r218518368) [](ancestors = 218518368,218508911)

Reply:
we decided to keep it at 0x00010002 

---
In reply to: [218504709](https://github.com/dotnet/machinelearning/pull/853#discussion_r218504709) [](ancestors = 218504709,218200584)